src/config/feeBySourceChain.ts

```diff
@@ -0,0 +1,12 @@
+import { ChainInfo } from "@axelar-network/axelarjs-sdk"
+
+export const feeBySourceChain = (sourceChain: ChainInfo) => {
+  if (!sourceChain) return ""
+
+  if (sourceChain?.module === "axelarnet") return "~2-5 minutes"
+
+  if (["ethereum", "polygon"].includes(sourceChain?.chainName.toLowerCase()))
+    return "~15-20 minutes"
+
+  return "~3-5 minutes"
```

Comment:
```suggestion
  return "~3 minutes"
```
Since it's indicated as approximate, no need for a range I think